--- /home/tstromberg/src/jquery-slug/jquery.slug.js	2012-08-19 22:35:14.277385745 -0700
+++ jquery.slug.js	2012-08-22 22:32:42.849988534 -0700
@@ -60,7 +60,8 @@
         'Ĳ': 'IJ',
         'ĳ': 'ij',
         'Œ': 'OE',
-        'ƒ': 'f'
+        'ƒ': 'f',
+        "'": '',
     };
     
     /**
@@ -71,10 +72,10 @@
     $.slug = function(string, replacement, map) {
         
         if($.type(replacement) == 'undefined') {
-            replacement = '_';
+            replacement = '-';
         } else if($.type(replacement) == 'object') {
             map = replacement;
-            replacement = '_';
+            replacement = '-';
         }
         
         transliteration['[^a-zA-Z0-9]'] = replacement;
@@ -93,7 +94,9 @@
             slug = slug.replace(re, value);
         });
         
-        return slug;
+        // clean up trailing and repeated underscores
+        slug = slug.replace(/^-+/, '').replace(/-+$/, '').replace(/-{2,}/, '-');
+        return slug.toLowerCase();
         
     };
     
